weather/src/test/java/com/weather/api/mapper/DateEntityMapperIntegrationTest.java

```diff
@@ -18,6 +19,8 @@
 
   @Autowired private DateMapper dateMapper;
 
+  private static final long SECS = 1563961730;
```

Comment:
Please multiply values instead of magic number.

F.e in 1 hour = 60 * 60 seconds